platform/machine/qemu/cluster.go

```diff
@@ -186,11 +181,23 @@ func (qc *Cluster) NewMachine(userdata *conf.UserData) (platform.Machine, error)
 			"-device", qc.virtio("9p", "fsdev=cfg,mount_tag=config-2"))
 	}
 
+	fdnum := 3
```

Comment:
nit: I would move these declarations down below the declaration of addDisk to make it clear they aren't being closed on by the closure.